crates/bevy_ui/src/camera_config.rs

```diff
@@ -11,7 +11,7 @@ use bevy_render::extract_component::ExtractComponent;
 /// When a [`Camera`] doesn't have the [`UiCameraConfig`] component,
 /// it will display the UI by default.
 ///
-/// [`Camera`]: bevy_render::camera::Camera
+/// [Camera]: bevy_render::camera::Camera
```

Comment:
```suggestion
```

I don't thing it is actually needed. `Camera` is in scope and should just work as an intra doc link